sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicLogicalOperators.scala

```diff
@@ -167,6 +212,8 @@ case class Intersect(left: LogicalPlan, right: LogicalPlan) extends SetOperation
 
     Statistics(sizeInBytes = sizeInBytes, isBroadcastable = isBroadcastable)
   }
+
+  override def sql: String = s"(${left.sql}) INTERSECT (${right.sql})"
```

Comment:
I think `INTERSECT` et al should be a value to later limit typos (I think there are tests that test that the output contains `INTERSET`, right?)